scripts/setup_irods.py

```diff
@@ -194,13 +194,20 @@ def check_hostname():
         raise IrodsError('The hostname (%s) must resolve to the local machine.' % (hostname))
 
 def determine_server_role(irods_config):
-    catalog_service_roles = set(['provider', 'consumer'])
+    # enum
```

Comment:
I think this needs more words. How about ...
```py
# The list of catalog service roles supported by the server.
```

Reply:
certainly